src/main/java/net/minecraftforge/gradle/tasks/dev/ChangelogTask.java

```diff
@@ -50,7 +48,7 @@ public class ChangelogTask extends DefaultTask {
     public void doTask() throws IOException {
         if (getAuthName() != null && getAuthPassword() != null) {
             String raw = getAuthName() + ":" + getAuthPassword();
-            auth = "Basic " + DatatypeConverter.printBase64Binary(raw.getBytes());
+            auth = "Basic " + new String(Base64.getEncoder().encode(raw.getBytes()));
```

Comment:
is it better to pass encoding? I think this makes environment dependent behavior

Reply:
I pass default charset for encoding is it good or any suggestions about charsets ?

Reply:
I think ASCII or  iso-8859-1 (a.k.a. Latin-1) is one good encoding. because they're minimal charset for ASCII.

Reply:
Now uses iso-8859-1